File: C/Exercices/secao05/exercicio03.c
#include <stdio.h>

int main()
{
    // var
    int vetor[10];

    // entradas
    for (int i = 0; i < 10; i++)
    {
        printf("Informe o valor para o vetor: ");
        scanf("%d", &vetor[i]);
    }
    for (int i = 9; i >= 0; i--)
    {
        printf("%d\n", vetor[i]);
    }
}

File: Java/src/secao08/Exercicio03.java
package secao08;

import java.util.Scanner;

public class Exercicio03 {
    public static void main(String[] args) {
        // var
        int[] vetor = new int[10];
        Scanner teclado = new Scanner(System.in);

        // processing
        for (int i = 0; i < 10; i++) {
            // input
            System.out.println("Informe um valor para o vetor: ");
            vetor[i] = teclado.nextInt();
        }
        for (int i = (vetor.length - 1); i >= 0; i--) {
            System.out.println(vetor[i]);
        }
        teclado.close();
    }
}
